ILD/compact/ILD_l5_v11/TrackerDiskModuleIn.xml

```diff
@@ -0,0 +1,27 @@
+<lccdd>
```

Comment:
This seems to be a copy(?) of `../ILD_common_v2/TrackerDiskModuleIn.xml`. Could we use that instead of creating a copy here?